homeassistant/components/camera/services.yaml

```diff
@@ -51,6 +51,22 @@ play_stream:
       description: (Optional) Stream format supported by media player.
       example: 'hls'
 
+record:
+  description: Record live camera feed.
+  fields:
+    entity_id:
+      description: Name of entities to record.
+      example: 'camera.living_room_camera'
+    filename:
+      description: Template of a Filename. Variable is entity_id.  Must be mp4.
+      example: '/tmp/snapshot_{{ entity_id }}.mp4'
+    duration:
+      description: (Optional) Target recording length (in seconds).
```

Comment:
What's the default?

Reply:
30 seconds - Should we add that it in the `services.yaml`?

Reply:
Yeah